From 9cebd7bb24de1e69c19c3cd6281cc3ef94113da6 Mon Sep 17 00:00:00 2001
From: Chris Mauzey <mauzey1@llnl.gov>
Date: Fri, 22 Sep 2023 10:23:04 -0700
Subject: [PATCH] Remove expat link for udunits2 from configure script

---
 configure    | 4 ++--
 configure.ac | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index b8cbfa2..f0aec1d 100755
--- a/configure
+++ b/configure
@@ -4905,9 +4905,9 @@ else $as_nop
 fi
 
    if  test ${RTAG} != "none"  ; then
-     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib  ${RTAG}${with_udunits2}/lib -ludunits2 -lexpat"
+     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib  ${RTAG}${with_udunits2}/lib -ludunits2"
    else
-     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib -ludunits2 -lexpat"
+     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib -ludunits2"
    fi
   else
     UDUNITS2FLAGS="-ludunits2"
diff --git a/configure.ac b/configure.ac
index cb011ba..b51a55c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -216,9 +216,9 @@ if [ test x${with_udunits2} != xyes ] ; then
     [UDUNITS2FLAGS=" -I${with_udunits2}/include/udunits2"],
     [UDUNITS2FLAGS=" -I${with_udunits2}/include"])
    if [ test ${RTAG} != "none" ] ; then
-     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib  ${RTAG}${with_udunits2}/lib -ludunits2 -lexpat"
+     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib  ${RTAG}${with_udunits2}/lib -ludunits2"
    else
-     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib -ludunits2 -lexpat"
+     UDUNITS2LDFLAGS=" -L${with_udunits2}/lib -ludunits2"
    fi
   else
     UDUNITS2FLAGS="-ludunits2"
-- 
2.39.2 (Apple Git-143)

